app/Http/Controllers/CommunityController.php

```diff
@@ -40,12 +41,17 @@ class CommunityController extends Controller
     */
     protected $section = 'community';
 
+    public function __construct()
+    {
+        $this->middleware('auth', ['only' => ['postSlackAgree']]);
+    }
```

Comment:
you forgot to call `parent::__construct()`, causing error when rendering the view.